### ISSUE TITLE
v-btn disabled not working if :to is used at the same time

### ISSUE THREAD

**Issue by jrast** [vuetifyjs/vuetify#958]
### Steps to reproduce
If a button has `:to` set, binding `:disabled` does not work.

As CodePen had Problems, I created a public gist here: https://gist.github.com/jrast/f09134a9dc5ecf1610c433c1cf4705a8

OK, CodePen is here: https://codepen.io/anon/pen/XgodEj

### Versions
Vuetify 0.13.1

### What is expected ?
The button should be rendered as disabled.

### What is actually happening ?
The button is rendered as active

### Reproduction Link
https://gist.github.com/jrast/f09134a9dc5ecf1610c433c1cf4705a8

And the CodePen: https://codepen.io/anon/pen/XgodEj

**Comment by jrast** [comment]
Ok, CodePen is working again. I added the links in the issue.

**Comment by jrast** [comment]
Seems like this is fixed in the latest dev version.

I just checked out the latest dev (`git descripe --tags` --> `v0.13.1-46-gc8b57a3`) and its working. It's working in a demo project I set up and in my application also.

The last change made to the relevant code seems to be https://github.com/vuetifyjs/vuetify/commit/e289861eaea46b8968727486dd4f01252dfc1ec0, however, I can't figure out why this should have fixed the issue...
